tests/NClient/NClient.Tests/ClientTests/FormUrlencodedClientTest.cs

```diff
@@ -14,54 +15,52 @@ namespace NClient.Tests.ClientTests
     public class FormUrlencodedClientTest : ClientTestBase<IFormUrlencodedClientWithMetadata>
     {
         [Test]
-        public async Task PostAsync_WithIntValue_ShouldReturnIntInBody()
+        public async Task PostAsync_WithIntValue_ShouldReturnOk()
         {
             const int id = 1;
             var keyValues = new Dictionary<string, object>
             {
                 [nameof(id)] = id
             };
-            using var api = FormUrlencodedApiMockFactory.MockPostMethod(id, keyValues);
+            using var api = FormUrlencodedApiMockFactory.MockPostMethod(keyValues);
 
             var result = await NClientGallery.Clients.GetRest().For<IFormUrlencodedClientWithMetadata>(host: api.Urls.First()).Build()
                 .PostAsync(id);
 
-            result.Should().Be(id);
+            result.StatusCode.Should().Be(StatusCodes.Status200OK);
         }
         
         [Test]
-        public async Task PostAsync_WithDictionary_ShouldReturnIntInBody()
+        public async Task PostAsync_WithDictionary_ShouldReturnOk()
         {
-            const int id = 1;
-            const int value = 1;
             var keyValues = new Dictionary<string, object>
             {
-                [nameof(id)] = id,
-                [nameof(value)] = value
+                ["id"] = 1,
```

Comment:
Why did you replace nameof() to strings?

Reply:
It doesn't matter ) Keys don't depend on variable names

Reply:
So... I asked about reasons of that changes. By my opinion, strings is harder for refactoring. (Yes, there are just two simple usage, but IT doesn't matter)

Reply:
So, if keys don't depend on variable names, why should I use `nameof`? ) Strictly speaking, `nameof` is little bit more complex than string, which means I made code simpler without losing quality

Reply:
Ok. I suggest you to use something one: nameof() or strings in every tests.

Reply:
Well, I've back `nameof` for uniformity

Reply:
Good job!